privacy-policy.ftd

```diff
@@ -168,15 +174,15 @@ The Company may use Personal Data for the following purposes:
 
 We may share Your personal information in the following situations:
 
--- list:
+-- tracking-list:
 -- list-item: - **With Service Providers:** We may share Your personal information with Service Providers to monitor and analyze the use of our Service, to contact You.
 
 -- list-item: - **For business transfers:** We may share or transfer Your personal information in connection with, or during negotiations of, any merger, sale of Company assets, financing, or acquisition of all or a portion of Our business to another company.
 -- list-item: - **With Affiliates:** We may share Your information with Our affiliates, in which case we will require those affiliates to honor this Privacy Policy. Affiliates include Our parent company and any other subsidiaries, joint venture partners or other companies that We control or that are under common control with Us.
 -- list-item: - **With business partners:** We may share Your information with Our business partners to offer You certain products, services or promotions.
 -- list-item: - **With other users:** when You share personal information or otherwise interact in the public areas with other users, such information may be viewed by all users and may be publicly distributed outside.
```

Comment:
```suggestion

-- list-item: - **With other users:** when You share personal information or otherwise interact in the public areas with other users, such information may be viewed by all users and may be publicly distributed outside.
```

Reply:
done!